scripts/clean_fcctr.sh

```diff
@@ -47,9 +47,18 @@ for d in `find /var/lib/cni/ -mindepth 1 -maxdepth 1  -type d | grep -v networks
     sudo rm -rf $d
 done
 
-echo Removing devmapper devices
-for de in `sudo dmsetup ls| cut -f1|grep snap`; do sudo dmsetup remove $de && echo Removed $de; done
-sudo dmsetup remove fc-dev-thinpool
+CONTAINERID=$(basename $(cat /proc/1/cpuset))
```

Comment:
what if I want to run this script on the bare host (no docker)?

Reply:
`basename $(cat /proc/1/cpuset) returns` "/" when executed on bare host. When this script is executed on bare host, thinpool device named "fc-dev-thinpool-snap-XX" is processed instead of "\<container id\>_thinpool-snap-XX".